src/coord/rmr/test/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 
-get_filename_component(root ${CMAKE_CURRENT_SOURCE_DIR}/../../../.. ABSOLUTE)
+get_filename_component(root ../../../.. ABSOLUTE)
 message("# coord/rmr/test: root: " ${root})
 
-include_directories(..)
```

Comment:
why not move all the tests into the root `tests` directory?

Reply:
In next PR